interoperator/config/samples/interoperator-config.yaml

```diff
@@ -0,0 +1,35 @@
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: interoperator-config
+data:
+  instanceWorkerCount: "5"
```

Comment:
Anyway we have to parse YAML for the deeper structs for `instanceContollerWatchList` and `bindingContollerWatchList`. Why not just bite the bullet and dump the whole YAML as a single key in the configmap?

Reply:
Updated.